packages/catalog-model/src/kinds/GroupEntityV1alpha1.ts

```diff
@@ -44,15 +44,35 @@ export class GroupEntityV1alpha1Policy implements EntityPolicy {
         .object({
           type: yup.string().required().min(1),
           parent: yup.string().notRequired().min(1),
-          ancestors: yup.array(yup.string()).required(),
-          children: yup.array(yup.string()).required(),
-          descendants: yup.array(yup.string()).required(),
+          // Use these manual tests because yup .required() requires at least
+          // one element and there is no simple workaround -_-
+          ancestors: yup.array(yup.string()).test({
+            name: 'isDefined',
+            message: 'ancestors must be defined',
+            test: v => Boolean(v),
+          }),
```

Comment:
can't do `.required()`, can't do `.required().min(0)`, can't do `.defined()` ... all for various reasons. Ughh.

There are ways to extend the yup commands to add like `.requiredOrEmpty()` but to make it play nice with TS you have to add a `.d.ts` file for it too, soooooo ... opted not to do that right now.